Training data: 100000
Validation data: 6002
3D Conv Out: (None, 50, 32, 32, 64)
3D Conv Out Reshape: (None, 32, 32, 64)
Resnet18 Out: (None, 1, 1, 512)
Resnet18 Linear Out: (None, 256)
Input to GRU: (None, 50, 256)
GRU Out: (None, 50, 29)
ResNet LSTM Pretrain weights loaded
outv: (None, 200, 256)
outa: (None, 200, 256)
hidden (None, 256)
dec (None, 1, 512)
Traceback (most recent call last):
  File "main_tasnet_attention.py", line 117, in <module>
    tasnet = TasNet(time_dimensions=200, frequency_bins=257, n_frames=50, attention=True, lipnet_pretrained=True,  train_lipnet=False)
  File "/data/AV-speech-separation1/models/tdavss_attention.py", line 75, in __init__
    self.build()
  File "/data/AV-speech-separation1/models/tdavss_attention.py", line 149, in build
    output,hidden_state, cell_state= decoder_recurrent_layer(decoder_input,initial_state=[hidden_state, cell_state])
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_core/python/keras/layers/recurrent.py", line 662, in __call__
    output = super(RNN, self).__call__(full_input, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_core/python/keras/engine/base_layer.py", line 817, in __call__
    self._maybe_build(inputs)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_core/python/keras/engine/base_layer.py", line 2141, in _maybe_build
    self.build(input_shapes)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_core/python/keras/layers/recurrent.py", line 554, in build
    self._validate_state_spec(state_size, self.state_spec)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_core/python/keras/layers/recurrent.py", line 591, in _validate_state_spec
    raise validation_error
ValueError: An `initial_state` was passed that is not compatible with `cell.state_size`. Received `state_spec`=ListWrapper([InputSpec(shape=(None, 256), ndim=2), InputSpec(shape=(None, 256), ndim=2)]); however `cell.state_size` is [512, 512]
